### PR TITLE
Fixes some basic issues of all tutorials

### PR DESCRIPTION
**Pull request by Gbahdeyboh**
Signed-off-by: Gbahdeyboh gbahdeybohbello@gmail.com

This PR fixes some basic issues across the whole tutorials. 

It fixes the following issues:

- #20 
- #18 
- #16 
- #17 
- #19

(_Adding more_)

### CODE REVIEW

**Comment by Gbahdeyboh** [comment]
@neelanjan00 @uditgaurav.

I have been taking a look at the Done button at the end of all the codelabs redirecting to a 404 page, but I've been unable to find a fix to it. It's a bit weird because it does not happen when I test the codelab using the Preview Code lab extension. It only happens after the code generation is done. 

I would have edited the code, but the"Done" button is not a part of the code. A huge chunk of the UI is injected by some codelabs JS script. 

I also had a look at the issues on the codelab repository, but couldn't quite find any issue around this specific problem. 

I think the best I can do now is to file a new issue on the codelabs repository and hopefully someone from their community fixes it.

**Comment by Gbahdeyboh** [comment]
@neelanjan00 @uditgaurav.

I have been taking a look at the Done button at the end of all the codelabs redirecting to a 404 page, but I've been unable to find a fix to it. It's a bit weird because it does not happen when I test the codelab using the Preview Code lab extension. It only happens after the code generation is done. 

I would have edited the code, but the"Done" button is not a part of the code. A huge chunk of the UI is injected by some codelabs JS script. 

I also had a look at the issues on the codelab repository, but couldn't quite find any issue around this specific problem. 

I think the best I can do now is to file a new issue on the codelabs repository and hopefully someone from their community fixes it.

**Comment by Gbahdeyboh** [comment]
@neelanjan00 can you lease review to see if the raised issues has been fixed?

File: tutorial-defining-steady-state-hypothesis/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <meta name="theme-color" content="#4F7DC9">
   <meta charset="UTF-8">
-  <title>Defining Steady State Hypothesis</title>
+  <title>xDefining Steady State Hypothesis</title>
```
Comment (neelanjan00) [inline tutorial-defining-steady-state-hypothesis/index.html:9]:
```suggestion
  <title>Defining Steady State Hypothesis</title>
```

File: tutorial-defining-steady-state-hypothesis/index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
   <google-codelab-analytics gaid="UA-49880327-14"></google-codelab-analytics>
   <google-codelab codelab-gaid=""
                   id=""
-                  title="Defining Steady State Hypothesis"
+                  title="xDefining Steady State Hypothesis"
```
Comment (neelanjan00) [inline tutorial-defining-steady-state-hypothesis/index.html:26]:
```suggestion
                  title="Defining Steady State Hypothesis"
```

**Review by neelanjan00 — CHANGES_REQUESTED** [review]

**Comment by neelanjan00** [comment]
LGTM, please fix these two typos then I will approve.

**Comment by neelanjan00** [comment]
You can also fix the DCO check @Gbahdeyboh

**Comment by Gbahdeyboh** [comment]
I'm not sure why the DCO is not passing. My commits are signed-off

**Pull request closed without merging**